pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
wait, I thought we were removing the requirement for the recycler to remount the volume, and documenting the requirement for `nfsv3 || adding delete permission || removing nfsv4 acl` in order to use this recycler